contracts/src/PriceFeedDirectory.sol

```diff
@@ -19,9 +19,16 @@ contract PriceFeedDirectory {
 
     event FeedAdded(address indexed aggregator, string appProviderChainPair);
 
-    // Maps the hashed and lowercase hex-encoded app ID (without leading 0x), price provider hostname, (optional) chain and trading pair (or contract address without leading 0x) separated by / to the data feed.
-    // Key examples:
-    // - keccak256("005a216eb7f450bcc1f534a7575fb33d611b463fa2/bitstamp.net/btc_usd")
+    // Maps the hashed and lowercase hex-encoded app ID (without leading 0x) and providerChainPair
+    // (which can be aggregated or per-provider) to the data feed.
+    //
+    // Key examples (aggregated multi-source feeds - recommended):
+    // - keccak256("005a216eb7f450bcc1f534a7575fb33d611b463fa2/aggregated/btc/usd")
+    // - keccak256("005a216eb7f450bcc1f534a7575fb33d611b463fa2/aggregated/eth/usd")
+    // - keccak256("005a216eb7f450bcc1f534a7575fb33d611b463fa2/aggregated/rose/usd")
+    //
+    // Legacy per-exchange or DEX formats (still accepted):
```

Comment:
I wouldn't mark direct exchange prices as legacy. Some dapps may deliberately want to rely on that data.

```suggestion
    // Direct per-exchange keys or DEX formats:
```